geonode/api/resourcebase_api.py

```diff
@@ -304,25 +306,27 @@ def build_haystack_filters(self, parameters):
                         bbox_right__lte=left))
 
         # Apply sort
-        if sort.lower() == "-date":
-            sqs = (
-                SearchQuerySet() if sqs is None else sqs).order_by("-date")
-        elif sort.lower() == "date":
-            sqs = (
-                SearchQuerySet() if sqs is None else sqs).order_by("date")
-        elif sort.lower() == "title":
-            sqs = (SearchQuerySet() if sqs is None else sqs).order_by(
-                "title_sortable")
-        elif sort.lower() == "-title":
-            sqs = (SearchQuerySet() if sqs is None else sqs).order_by(
-                "-title_sortable")
-        elif sort.lower() == "-popular_count":
-            sqs = (SearchQuerySet() if sqs is None else sqs).order_by(
-                "-popular_count")
-        else:
-            sqs = (
-                SearchQuerySet() if sqs is None else sqs).order_by("-date")
-
+        # if sort.lower() == "-date":
```

Comment:
In our instance, we don't use the Haystack. Unfortunately, i don't know the behaviour with the Haystack, and i can't tell if it we lose sorting capabilities when using it.